### PR TITLE
Set up the project file

### PR DESCRIPTION
**Pull request by Managram**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
+
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+
+Capybara.app = MakersBnb
+
+RSpec.configure do |config|
+
```
Comment (junyuanxue) [inline spec/spec_helper.rb:12]: the space is so ugly man

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  config.include Capybara::DSL
+
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+
```
Comment (junyuanxue) [inline spec/spec_helper.rb:22]: same here!

**Comment by junyuanxue** [comment]
:shipit:

**Comment by Managram** [comment]
closes #2

**Pull request merged**